File: momentra/momentra/Shell/PersonalEmpty/PersonalExpenseSheet.swift
import SwiftUI

struct PersonalExpenseSheet: View {
    let momentId: String
    @Binding var isPresented: Bool
    var pulseFamily: PersonalPulseFamily = .lifeOperations
    var onSaved: () -> Void

    var body: some View {
        PersonalMasterExpenseSheet(
            momentId: momentId,
            pulseFamily: pulseFamily,
            onClose: { isPresented = false },
            onSaved: onSaved
        )
    }
}

File: apk/app/src/main/java/com/example/momentra/ui/shell/personal/shared/PersonalExpenseSheet.kt
package com.example.momentra.ui.shell.personal.shared

import androidx.compose.runtime.Composable
import com.example.momentra.data.repository.PersonalSliceRepository

/** Thin wrapper — delegates to [PersonalMasterExpenseSheet] (Figma `453:9376`). */
@Composable
fun PersonalExpenseSheet(
    momentId: String,
    visible: Boolean,
    pulseFamily: PersonalPulseFamily = PersonalPulseFamily.LIFE_OPERATIONS,
    onDismiss: () -> Unit,
    onSaved: () -> Unit,
    repository: PersonalSliceRepository = PersonalSliceRepository(),
) {
    PersonalMasterExpenseSheet(
        momentId = momentId,
        visible = visible,
        pulseFamily = pulseFamily,
        onDismiss = onDismiss,
        onSaved = onSaved,
        repository = repository,
    )
}
